README.md

```diff
@@ -41,6 +41,13 @@ To configure the zoom and pan plugin, you can simply add new config options to y
 		// Enable drag-to-zoom behavior
 		drag: true,
 
+		// Drag-to-zoom rectangle style can be customized
+		// drag: {
```

Comment:
no need to have these lines commented out

Reply:
Fixed